cg/models/demultiplex/flow_cell.py

```diff
@@ -210,10 +211,19 @@ def validate_flow_cell_dir_name(self) -> None:
             LOG.warning(message)
             raise FlowCellError(message)
 
-    def is_demultiplexing_started(self) -> bool:
-        """Check if demultiplexing started path exists."""
+    def has_demultiplexing_started_locally(self) -> bool:
+        """Check if demultiplexing has started path exists on the cluster."""
         return self.demultiplexing_started_path.exists()
 
+    def has_demultiplexing_started_on_deck(self) -> bool:
```

Comment:
```suggestion
    def has_demultiplexing_started_on_sequencer(self) -> bool:
```
more intuitive on location like this